src/uct/tcp/sockcm/sockcm_cm.c

```diff
@@ -0,0 +1,94 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include "sockcm_cm.h"
+
+
+ucs_config_field_t uct_sockcm_cm_config_table[] = {
```

Comment:
could be static?

Reply:
it's defined here but set in sockcm_md.c

Reply:
ok